{{ cookiecutter.project_slug }}/README.rst

```diff
@@ -1,22 +1,29 @@
-# {{ cookiecutter.project_desc }}
+===============================
+{{ cookiecutter.project_desc }}
+==============================
```

Comment:
Probably we will have to fix this for every project_desc lenght, but at least on the template, they should match the lenght from before and after

Reply:
Good catch,
I think that we can do something like:

```
{{ "=" * ( cookiecutter.project_desc | count) }}
{{ cookiecutter.project_desc }}
{{ "=" * ( cookiecutter.project_desc | count) }}
```

Reply:
Changed and tested locally, it works